### PR TITLE
Product page added

### PR DESCRIPTION
**Pull request by DmytroPetrenko**

### CODE REVIEW

File: src/components/DescriptionList.vue [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<template>
+	<dl>
+		<div v-for="(value, name) in list" :key="name" class="listPair">
```
Comment (angryParhan) [inline src/components/DescriptionList.vue:3]: `:key="name"`? Could a several names be the same?

File: src/components/DescriptionList.vue [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+			<dt class="characteristics-full__label">
+				<span>{{ name }}</span>
+			</dt>
+			<dd class="characteristics-full__value">{{ value }}</dd>
+		</div>
+	</dl>
+</template>
+<script>
+export default {
+	props: { list: Object },
```
Comment (angryParhan) [inline src/components/DescriptionList.vue]: wrong props declaration!

File: src/views/ProductPage.vue [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+										{{ $t("product.contributed_by") }}
+										{{ activeProduct.contributed_by }}
+									</p>
+								</v-card>
+							</v-tab-item>
+						</v-tabs-items>
+
+						<!-- <h3 class="mmat-5">{{ $t("product.characteristics") }}</h3>
+						<DescriptionList class="mmat-2" :list="productDescriptionObj" /> -->
+					</v-col>
```
Comment (angryParhan) [inline src/views/ProductPage.vue:59]: do we need this?

File: src/views/ProductPage.vue [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+					disabled: true,
+					to: { name: routeNames.product, params: this.$route.params.id },
+				},
+			]
+		},
+		productDescriptionObj() {
+			let objCopy = Object.assign({}, this.activeProduct)
+			objCopy = this.spliceOnObj(objCopy, 6, 8)
+			return objCopy
+		},
```
Comment (angryParhan) [inline src/views/ProductPage.vue:119]: I don't understand what are you doing here could you rework logic in a more appropriate way?

File: src/views/ProductPage.vue [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+	},
+	methods: {
+		...mapActions("products", ["fetchProductById", "setActiveProduct"]),
+		buy_btnHandler() {
+			console.log("buy_btn handler")
+		},
+		spliceOnObj(object, number, number2) {
```
Comment (angryParhan) [inline src/views/ProductPage.vue]: object, number, number2 - are not good variable names

File: src/views/ProductPage.vue [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+			if (!number2) {
+				number2 = 1
+			}
+			Object.entries(object).forEach(function ([key, item], index) {
+				if (index >= number && index - number < number2) {
+					newObj[key] = item
+				}
+			})
+			return newObj
+		},
```
Comment (angryParhan) [inline src/views/ProductPage.vue]: could you explain why do you need this method?

File: src/components/DescriptionList.vue (hunk 1 above)
Comment (DmytroPetrenko) [reply src/components/DescriptionList.vue:3]: no, it is names of properties

File: src/views/ProductPage.vue (hunk 4 above)
Comment (DmytroPetrenko) [reply src/views/ProductPage.vue:119]: I write part of props from activeProduct object to new one

Comment (DmytroPetrenko) [reply src/views/ProductPage.vue:119]: do not find a better way to do the same

File: src/components/Card.vue [hunk 7]
```diff
@@ -35,25 +33,23 @@ import { mapActions, mapGetters } from "vuex"
...
+	props: ["item"],
 	computed: {
 		...mapGetters("products", ["isFavourites"]),
+		icon() {
+			if (this.isFavourites(this.item.id)) {
+				return "mdi-star"
+			} else {
+				return "mdi-star-outline"
+			}
+		},
```
Comment (angryParhan) [inline src/components/Card.vue:45]: `return this.isFavourites(this.item.id) ? "mdi-star" : "mdi-star-outline"`

File: src/components/DescriptionList.vue [hunk 8]
```diff
@@ -10,7 +10,7 @@
 </template>
 <script>
 export default {
-	props: { list: Object },
+	props:  ["list"],
```
Comment (angryParhan) [inline src/components/DescriptionList.vue]:
```
not in this way the right way is
list: {
  type: Object,
  ...(here required: true or default Object value)
}
```

File: src/views/ProductPage.vue [hunk 9]
```diff
@@ -101,8 +98,23 @@ export default {
...
+				"abv",
+				"ibu",
+				"target_fg",
+				"target_og",
+				"ebc",
+				"srm",
+				"ph",
+				"attenuation_level",
+			]
+			let objCopy = {}
```
Comment (angryParhan) [inline src/views/ProductPage.vue]: `const objCopy = {}`

**Pull request merged**